.gitmodules

```diff
@@ -77,7 +77,7 @@
 [submodule "grafana-dashboards"]
 	path = sources/grafana-dashboards
 	url = https://github.com/percona/grafana-dashboards
-	branch = PMM-2.0
+	branch = add-instance
```

Comment:
Branch name should start with a ticket number.

Reply:
Done